gateway/1.10/modules/ROOT/pages/flex-gateway-managed-ingress-egress.adoc

```diff
@@ -0,0 +1,33 @@
+= Deploying Agent Network Ingress and Egress Managed Flex Gateways 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Agent Network deployments require both an ingress and egress Flex Gateway. The ingress Flex Gateway sits in front of broker and API endpoints to manage and enforce policies on traffic entering the network. The egress Flex Gateway sits on outbound paths from brokers and agents to external agents and enforces policies, manages connections, and emits telemetry data.
+
+The only functional difference between ingress and egress Flex Gateways, is that the egress Gateway doesn't have a public endpoint. Only clients inside the same private space as the egress gateway can make requests to it. Traffic leaving the private space to external services passes through the egress gateway. 
+
+image::agent-fabric-architecture.png["Agent Network diagram showing agents and MCP servers defined in YAML and published to Exchange, with traffic through Flex Gateway, governed by API Manager, and observed in Monitoring and Visualizer"]
+
+NOTE: For a more detailed description of this architecture diagram, see xref:agent-fabric::index.adoc#architecture[Agent Fabric Architecture].
+
+== Deploy Ingress and Egress Managed Flex Gateways
+
+The easiest way to deploy your Agent Network ingress and egress Flex Gateways is by using Anypoint Code Builder or Anypoint Command Line Interface (CLI):
```

Comment:
(for later: Look at FG capitalization and (I think) make lowercase unless referring to the product name - so "flex gateways")